swagger.json

```diff
@@ -1129,7 +1129,7 @@
               ],
               "summary": "/fleet/drivers/{driver_id:[0-9]+}/log_edits",
               "description": "Get a summary of the carrier edits for a specified driver. Note: This does not include edits made by driver",
```

Comment:
nit: Could you add a period at the end of this description?

Reply:
Oh sure!